DiamondLightSource/python-artemis
# Calculations and run plan for OAV loop  centring

Fixes #420 
Replaces #453, since that branch contained other changes.
To test:
* Run against s03 and the branch of issue 26 in 
* Check output is acceptable.
